pyanaconda/ui/gui/spokes/custom.py

```diff
@@ -314,8 +314,7 @@ def _initialize(self):
             obj = cls()
 
             # btrfs is always handled by on_device_type_changed
-            supported_fs = (obj.type != "btrfs" and
-                            obj.type != "tmpfs" and
+            supported_fs = (obj.type not in ("btrfs", "tmpfs", "ntfs") and
```

Comment:
Just a late note on this -- might be worth to have the list stored somewhere else with an appropriate name like ``UNSUPPORTED_FILESYSTEMS``.